plutus-ir/test/serialization/serializeTestTerm.plc

```diff
@@ -0,0 +1,93 @@
+(let
```

Comment:
This also looks like it's the compiled version, not the version with datatypes.

Reply:
I can solve this mystery: serializeTestTerm.plc.golden is actually never used in testing, which is also why (as @effectfully points out above), the file uses `fix` and `wrap` instead of `ifix` and `iwrap` but this didn't trigger a parse error. I just blindly copied all the `.plc.golden` tests as `.plc` files and didn't realize this.